src/zarr/fixture/.zgroup

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I believe this was added in error

Reply:
ah right, this is the zombie fixture directory that gets created by tests, see https://github.com/zarr-developers/zarr-python/pull/1813